src/cloud-api-adaptor/test/provisioner/trustee_kbs.go

```diff
@@ -127,6 +127,18 @@ func NewKeyBrokerService(clusterName string, cfg *envconf.Config) (*KeyBrokerSer
 
 	}
 
+	customPCCSURL := os.Getenv("CUSTOM_PCCS_URL")
+	if customPCCSURL != "" {
+		log.Info("CUSTOM_PCCS_URL is provided, write custom PCCS config")
+		configFilePath := filepath.Join(TRUSTEE_REPO_PATH, "/kbs/config/kubernetes/custom_pccs/sgx_default_qcnl.conf")
+		collateralUrl := "https://api.trustedservices.intel.com/sgx/certification/v4/"
+		config := fmt.Sprintf(`{ "pccs_url": "%s", "collateral_service": "%s"}`, customPCCSURL, collateralUrl)
```

Comment:
I don't think the `collateral_service` is required, so it can be dropped.

I am not sure if you'll need ` "use_secure_cert": false`, if the tests pass as they're, you're good to go without that.

Reply:
FWIW, for quote verification (which is the only thing we are doing with Trustee here), only `collateral_service` is needed and for that we can use the same [Azure setting](https://learn.microsoft.com/en-us/azure/security/fundamentals/trusted-hardware-identity-management#how-do-i-use-intel-qpl-with-trusted-hardware-identity-management)